packages/core/src/Asset/cip67.ts

```diff
@@ -0,0 +1,73 @@
+import { Cardano } from '..';
+import { InvalidArgumentError, OpaqueNumber } from '@cardano-sdk/util';
+import crc8 from './crc8';
+
+const ASSET_LABEL_LENGTH = 8;
+const ASSET_LABEL_BRACKET = '0';
+
+export enum AssetNameLabelNum {
+  ReferenceNFT = 100,
+  SubStandard222 = 222,
+  SubStandard333 = 333,
+  SubStandard444 = 444
```

Comment:
Let's use more descriptive names. These labels are better explained in [cip68](https://cips.cardano.org/cips/cip68/)
```suggestion
  UserNFT = 222,
  UserFT = 333,
  UserRFT = 444
```